package.json

```diff
@@ -4,10 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "node ./src/bin/www",
+    "start": "set DEBUG=dev && node ./src/bin/www",
```

Comment:
Please move this `DEBUG=dev` to the `devstart` script

Reply:
Ok noted